hack/ci/env.sh

```diff
@@ -27,3 +27,5 @@ IP=$(local_ip)
 export KUBE_MASTER_URL=$IP
 export KUBE_MASTER_IP=$IP
 export KUBE_MASTER=$IP
+
+alias k=kubectl
```

Comment:
This seems unused, or was this added for debugging purposes (when you logged into the VM). If so, let's add a comment about it.